### ISSUE TITLE
Desktop Wallet not working

### ISSUE THREAD

**Issue by farinspace** [LiskHQ/lisk-desktop#5598]
### Expected behavior
Desktop wallet shows latest blockchain transactions and the latest blocks on mainnet.

### Actual behavior
However, only transactions from May 21, 2024 and prior are displayed. No blocks are displayed since may 21, 2024 (no new blocks are displayed).

### Steps to reproduce
Install latest version of Desktop wallet.

### Which version(s) does this affect? (Environment, OS, etc...)
I have installed 3.0.3 on mac os and 3.0.3/3.0.2 on windows both with same results above.

-----

Maybe I am missing something here, initial impressions seems like the LISK L1 Mainnet simply isn't working any more ... I am trying to transfer LISK L1 tokens from Exodus to the LISK desktop wallet so that I can then convert to L2 tokens via wallet connect.

**Comment by farinspace** [comment]
The end goal is to use the lisk migration tool to claim and transfer tokens:

1) I have exported my LISK private key from Exodus wallet
2) I have used the following: https://github.com/przemerr/exodusRecovery to create a recovery file for import into lisk-desktop
3) I am able to recover using lisk-desktop ... the desktop wallet shows my balance.

And, the claim tool seems to work, it recognizes my balance, however when I go to sign a message lisk-desktop silently fails (enter password, click continue ... nothing happens).

Signing a plain message from the "recovered account" fails ... but signing a plain message from a newly created account seems to work.

I want to try to understand what is going on and if there is anything that can be done.

Thank you.
